proto/storage/process_listening_on_port.proto

```diff
@@ -60,4 +61,6 @@ message ProcessListeningOnPortStorage {
     ProcessIndicatorUniqueKey process = 7;
 
     string deployment_id = 8 [(gogoproto.moretags) = 'search:"Deployment ID,store" sql:"fk(Deployment:id),no-fk-constraint,index=btree,type(uuid)"'];
+
```

Comment:
nit:  It would be nice if this proto was lined up and formatted like most the others for readability.

Reply:
Done